app.js

```diff
@@ -9,9 +11,12 @@ var cookieParser = require('cookie-parser');
 var loginRouter = require('./routes/login');
 var patientRegistrationRouter = require('./routes/patient_registration');
 var doctorRegistrationRouter = require('./routes/doctor_registration');
-
+var index = require('./routes/login');
+var dash = require('./routes/dashboard');
+var logout = require('./routes/logout');
 const app = express();
 var router = express.Router();
+const publicdir = path.join(__dirname, './public');
```

Comment:
what is this for?

Reply:
just a variable to use public directly, didn't use it i think.